sphinx_immaterial/external_resource_cache.py

```diff
@@ -81,14 +81,28 @@ def get_cache_dir(app: sphinx.application.Sphinx) -> str:
     return getattr(app.config, _RESOURCE_CONFIG_KEY)
 
 
+def _config_inited(
+    app: sphinx.application.Sphinx, config: sphinx.config.Config
+) -> None:
+    """Inject path to local cache of mathjax dist.
+    Sphinx machinery will do the rest of the work for us."""
+    if "mathjax_path" in config:
+        config["mathjax_path"] = "tex-mml-chtml.js"
```

Comment:
I have to doube check this. I think this will override the user-defined path, but I'm not sure if it is a guarantee that mathjax is being used.

Reply:
Looks like this path will always be present if the mathjax ext is added to the conf.py `extensions`. Resolving this.